src/site/shared/containers/SelectionPopup/modules/Module.tsx

```diff
@@ -98,8 +99,17 @@ export const CreateModule = (<T extends any[], P extends ModuleTypes>(props: Mod
         return v;
     }
 
+    const checkType = (s: Selectable, Type: Function & { prototype: T[number]; }) => {
+        // check if s instanceof Type and also not instnace of anything in config.exclude
```

Comment:
nit: capitalize first letter of comment, also typo on `instance`, also mention this was necessary for issue #820